tests/store/artifact/test_databricks_artifact_repo.py

```diff
@@ -197,22 +197,21 @@ def test_log_artifact_azure(
         self, databricks_artifact_repo, test_file, artifact_path, expected_location
     ):
         with mock.patch(
-            DATABRICKS_ARTIFACT_REPOSITORY + "._get_write_credentials"
-        ) as write_credentials_mock, mock.patch(
+            DATABRICKS_ARTIFACT_REPOSITORY + "._get_write_credential_infos"
+        ) as write_credential_infos_mock, mock.patch(
             DATABRICKS_ARTIFACT_REPOSITORY + "._azure_upload_file"
         ) as azure_upload_mock:
-            mock_credentials = ArtifactCredentialInfo(
+            mock_credential_info = ArtifactCredentialInfo(
                 signed_uri=MOCK_AZURE_SIGNED_URI, type=ArtifactCredentialType.AZURE_SAS_URI
             )
-            write_credentials_response_proto = GetCredentialsForWrite.Response(
-                credentials=mock_credentials
-            )
-            write_credentials_mock.return_value = write_credentials_response_proto
+            write_credential_infos_mock.return_value = [mock_credential_info]
             azure_upload_mock.return_value = None
             databricks_artifact_repo.log_artifact(test_file.strpath, artifact_path)
-            write_credentials_mock.assert_called_with(MOCK_RUN_ID, expected_location)
+            write_credential_infos_mock.assert_called_with(
+                run_id=MOCK_RUN_ID, paths=[expected_location]
+            )
             azure_upload_mock.assert_called_with(
-                mock_credentials, test_file.strpath, expected_location
+                mock_credential_info, test_file.strpath, expected_location
```

Comment:
This change, and many others of the same form, just accommodate the renaming of `_get_write_credentials` and `_get_read_credentials_` to `_get_write_credential_infos` and `_get_read_credential_infos`, respectively, as well as the adjustment of these methods' return types from `GetCredentialsForRead.Response` and `GetCredentialsForWrite.Response` to lists of `ArtifactCredentialInfo` objects.